README.md

```diff
@@ -21,17 +21,17 @@ EvalML is an AutoML library which builds, optimizes, and evaluates machine learn
 ```shell
 pip install evalml
 ```
-#### Install with Facebook's Prophet (time series)
+#### Add-ons
```

Comment:
I'm super nit-picking but can we use ### instead? Seems like there's not a big difference between H4 and the bold below :)

Reply:
Fixed here
- https://github.com/alteryx/evalml/pull/2627/commits/13227609ed086d39a952912b9e5611525d1f6381

I also changed from bold to H4 headers for the subsections.